<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QTabWidget" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Booking Information</string>
  </property>
  <property name="currentIndex">
   <number>1</number>
  </property>
  <widget class="QWidget" name="flug">
   <attribute name="title">
    <string>Flugbuchung</string>
   </attribute>
   <widget class="QWidget" name="layoutWidget_8">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QLineEdit" name="flugAb"/>
     </item>
     <item>
      <widget class="QLineEdit" name="flugAn"/>
     </item>
    </layout>
   </widget>
   <widget class="QDoubleSpinBox" name="flugPreis">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_9">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>52</y>
      <width>63</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>von</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>bis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Preis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>Abfahrt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>Ankunft </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>Airline</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="flugID">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_10">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_12">
     <item>
      <widget class="QDateTimeEdit" name="flugVon"/>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="flugBis"/>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="flugFirm">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>400</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="car">
   <attribute name="title">
    <string>Autobuchung</string>
   </attribute>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLineEdit" name="autoAbhol"/>
     </item>
     <item>
      <widget class="QLineEdit" name="autoZurueck"/>
     </item>
    </layout>
   </widget>
   <widget class="QDoubleSpinBox" name="autoPreis">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>52</y>
      <width>63</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>von</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>bis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Preis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Abholen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Zur√ºck</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Firma</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="autoID">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_7">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QDateTimeEdit" name="autoVon"/>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="autoBis"/>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="autoFirm">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>400</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="hotel">
   <attribute name="title">
    <string>Hotelbuchung</string>
   </attribute>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLineEdit" name="hotelName"/>
     </item>
     <item>
      <widget class="QLineEdit" name="hotelTown"/>
     </item>
    </layout>
   </widget>
   <widget class="QDoubleSpinBox" name="hotelPreis">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>52</y>
      <width>63</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>von</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>bis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Preis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Hotel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Town</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="hotelID">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QDateTimeEdit" name="hotelVon"/>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="hotelBis"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="zug">
   <attribute name="title">
    <string>Zugbuchung</string>
   </attribute>
   <widget class="QLineEdit" name="zugID">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="zugPreis">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QDateTimeEdit" name="zugVon"/>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="zugBis"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>52</y>
      <width>63</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>von</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>bis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Preis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Abfahrt </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Ankunft</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>√ºber</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="zugAb"/>
     </item>
     <item>
      <widget class="QLineEdit" name="zugAn"/>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="zugUeber">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>400</y>
      <width>401</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
